ang/civiawards/award-creation/directives/review-panels/review-panels.directive.js

```diff
@@ -287,51 +289,70 @@
       var reviewPanelDataCopied = _.clone(reviewPanelData);
 
       _.each(reviewPanelDataCopied, function (reviewPanel) {
-        if (!reviewPanel.contact_settings) {
-          return;
-        }
+        reviewPanel.formattedContactSettings = formatContactSettings(reviewPanel);
+        reviewPanel.formattedVisibilitySettings = formattedVisibilitySettings(reviewPanel);
+      });
 
-        reviewPanel.formattedContactSettings = {
-          include: [], exclude: [], relation: []
-        };
+      return reviewPanelDataCopied;
+    }
 
-        _.each(reviewPanel.contact_settings.include_groups, function (includeGroupID) {
-          reviewPanel.formattedContactSettings.include.push(
-            groupsIndexed[includeGroupID].title
-          );
-        });
+    /**
+     * Format Review panel visibility settings
+     *
+     * @param {object} reviewPanel review panel object
+     * @returns {object} formatted visibility settings
+     */
+    function formattedVisibilitySettings (reviewPanel) {
+      var formattedVisibilitySettings = {
+        applicationStatus: reviewPanel.visibility_settings.application_status.map(function (status) {
+          return CaseStatus.getAll()[status].label;
+        }),
+        tags: reviewPanel.visibility_settings.application_tags.map(function (tagId) {
+          return tagsIndexed[tagId].name;
+        })
+      };
 
-        _.each(reviewPanel.contact_settings.exclude_groups, function (excludeGroupID) {
-          reviewPanel.formattedContactSettings.exclude.push(
-            groupsIndexed[excludeGroupID].title
-          );
-        });
+      return formattedVisibilitySettings;
+    }
 
-        _.each(reviewPanel.contact_settings.relationship, function (relationship) {
-          var specificRelationDetails = { relationshipLabel: '', contacts: [] };
+    /**
+     * Format Review panel contact settings
```

Comment:
Please include the kind of formatting that is applied to the review panel's contact settings.